conda_smithy/cli.py

```diff
@@ -54,62 +98,87 @@ class Init(Subcommand):
     subcommand = 'init'
     def __init__(self, parser):
         # conda-smithy init /path/to/udunits-recipe ./
-        subcommand_parser = Subcommand.__init__(self, parser, "Create a feedstock git repository.")
-        subcommand_parser.add_argument("recipe_directory")
-        subcommand_parser.add_argument("--feedstock-directory",
-                                       default='./{package.name}-feedstock')
-        subcommand_parser.add_argument("--no-git-repo", action='store_true',
-                                       default=False)
+
+        super(Init, self).__init__(parser, "Create a feedstock git repository, which can contain "
+                                           "one or multiple conda recipes.")
+        scp = self.subcommand_parser
+        scp.add_argument("recipe_directory", help="The path to the source recipe directory.")
+        scp.add_argument("--multi", action='store_true', default=False,
```

Comment:
you are reviewing the multi PR (which includes the other PR) :-)

-> You probably want to start with https://github.com/conda-forge/conda-smithy/pull/56